src/components/TextField/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+type TextFieldProps = {
+    label?: React.ReactNode;
+    placeholder?: string;
+    value?: string;
+    onChange?: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+    error?: boolean | string;
+    disabled?: boolean;
+    className?: string;
+}
+
+const TextField = ({
```

Comment:
```suggestion
const TextArea = ({
```
I think it should be textarea everywhere- component name, props name, file name, classnames etc.